### PR TITLE
Users model

### PR DESCRIPTION
**Pull request by ezhenezhen**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,7 +19,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.2'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
-
+gem 'rubocop', '~> 0.51.0', require: false
 # Use jquery as the JavaScript library
```
Comment (ezhenezhen) [inline Gemfile]: We dont need rubocop in production.
Move to development group

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
+class UsersController < ApplicationController
+  before_action :set_person, only: [:show, :edit, :destroy, :update]
+  skip_before_action :authenticate_user!, only: [:index, :show]
```
Comment (ezhenezhen) [inline app/controllers/users_controller.rb]: Why skip authentication?

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    respond_to do |format|
+      if @user.update(user_params)
+        format.html { redirect_to @user, notice: "Данные пользователя #{@user.nickname} успешно обновлены" }
+      else
+        format.html { render action: 'edit' }
+      end
+    end
+  end
+
+  def edit; end
```
Comment (ezhenezhen) [inline app/controllers/users_controller.rb:33]: Why nothing is here?

File: app/views/products/index.html.erb [hunk 4]
```diff
@@ -32,4 +32,10 @@
 
 <br>
 
-<%= link_to 'Создать новый продукт', new_product_path %>
+<%= link_to 'Создать новый продукт', new_product_path %> 
+<p></p>
+<% if current_user.nil? %>
+  <%= link_to new_user_session_path, class: 'login-button' do %>Sign in<% end %>
+<% else %>
```
Comment (ezhenezhen) [inline app/views/products/index.html.erb]: This have to move to general layout, so its available all over the site

File: config/locales/devise.en.yml [hunk 5]
```diff
@@ -0,0 +1,64 @@
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+en:
+  devise:
+    confirmations:
```
Comment (ezhenezhen) [inline config/locales/devise.en.yml:5]: Do we need russian translations?

File: test/controllers/users_controller_test.rb [hunk 6]
```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
```
Comment (ezhenezhen) [inline test/controllers/users_controller_test.rb:3]: Are tests passing?

File: Gemfile (hunk 1 above)
Comment (olindos) [reply Gemfile]: moved

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (olindos) [reply app/controllers/users_controller.rb]: I did as a test, deleted

File: app/views/products/index.html.erb (hunk 4 above)
Comment (olindos) [reply app/views/products/index.html.erb]: Moved in general layout

File: config/locales/devise.en.yml (hunk 5 above)
Comment (olindos) [reply config/locales/devise.en.yml:5]: add russian translations

File: test/controllers/users_controller_test.rb (hunk 6 above)
Comment (olindos) [reply test/controllers/users_controller_test.rb:3]: No, the test does not work.

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (olindos) [reply app/controllers/users_controller.rb:33]: there's not action,

Comment (ezhenezhen) [reply app/controllers/users_controller.rb:33]: So maybe just delete it?

**Pull request merged**